src/http.rs

```diff
@@ -252,6 +245,245 @@ impl HttpClient {
     }
 }
 
+#[cfg(feature = "async")]
+use futures::prelude::*;
+
+#[cfg(feature = "async")]
+impl HttpClient {
+    pub async fn tx_log(&self, actions: Vec<Action>) -> impl Future<Output = TxLogResponse> + Send {
```

Comment:
Shouldn't the Future Output be `Result<TxLogResponse, CruxError>`?

Reply:
Its a bug! need more time to fix it